roles/nodes/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: "Managing Storage Protect Nodes"
```

Comment:
Rename to 'Registering storage protect client node with schedule'

Reply:
No, its more generic than that